t/full-stack.t

```diff
@@ -118,13 +118,13 @@ sub check_scheduled_job_and_wait_for_free_worker ($worker_class) {
         }
     }
     Test::More::fail "no worker with class $worker_class showed up after $elapsed seconds";
-    Test::More::diag explain 'free workers: ', [map { $_->info } @$free_workers];
+    Test::More::always_explain 'free workers: ', [map { $_->info } @$free_workers];
     return ($relevant_jobs, $elapsed);
 }
 
 sub show_job_info ($job_id) {
     my $job = $schema->resultset('Jobs')->find($job_id);
-    Test::More::diag explain 'job info: ', $job ? $job->to_hash : undef;
+    Test::More::always_explain 'job info: ', $job ? $job->to_hash : undef;
```

Comment:
```suggestion
    Test::More::diag always_explain 'job info: ', $job ? $job->to_hash : undef;
```

Reply:
actually I removed the module prefix is `always_explain` is unique now